docs/integration_resources/resources/ftrack.md

```diff
@@ -0,0 +1,5 @@
+# Ftrack
+
+Shift catalogs and plugins for Ftrack are on the works!
+
+If you want to go ahead and give a try to our current development for Ftrack, please let us know through our contact page.
```

Comment:
Should we add here a link to the contact page? This question also applies to the Resolve and Deadline pages.

Reply:
Done